cli.js

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
+
+import './packages/autocannon-ui-backend/app.js'
```

Comment:
It would be better if we could have some control in this module instead of keeping all the logic in the backend package. why don't we move the logic to start the server here instead?

Reply:
Let me try it